keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java

```diff
@@ -132,6 +133,15 @@ public Response executeActionsEmail(@QueryParam("lifespan") Integer lifespan, Li
                     ErrorResponse.error("User is disabled", Status.BAD_REQUEST));
         }
 
+        RealmModel themeRealm = realm;
```

Comment:
Anyone can ask for any theme, do we have a way to restrict it ?

Reply:
No way to restrict this at Keycloak level... it will be done on bridge's side